cmd/check.go

```diff
@@ -243,14 +243,35 @@ func getExportFormats(executing string) []controldisplay.CheckExportFormat {
 	formats := []controldisplay.CheckExportFormat{}
 	exports := viper.GetStringSlice(constants.ArgExport)
 	for _, export := range exports {
+		if len(strings.TrimSpace(export)) == 0 {
+			// if this is an empty string, ignore
+			continue
+		}
+
 		parts := strings.SplitN(export, ":", 2)
-		format := parts[0]
-		fileName := generateDefaultExportFileName(format, executing)
-		if len(parts) > 1 {
-			if f, err := helpers.Tildefy(parts[1]); err == nil {
+
+		format := ""
+		fileName := ""
+
+		if len(parts) == 2 {
+			// we have two distinct parts - life is good
+			format = parts[0]
+			fileName = parts[1]
+			if f, err := helpers.Tildefy(fileName); err == nil {
```

Comment:
what happens in case of error?

Reply:
We keep it as it is and try to work with it.

If the `fileName` is not resolvable, `os` will error out when we try to create it

Reply:
yeah I don't like that really - got a screen shot of how that looks?

I think I'd rather handle it explicitly

If it is to remain implicit, we need a comment